collectors/bzimport/convertors.py

```diff
@@ -623,6 +623,28 @@ def requires_summary(self):
 
         return pairs.get(status)
 
+    @cached_property
+    def nist_cvss_validation(self):
+        """
+        Set Flaw field nist_cvss_validation from bugzilla flag nist_cvss_validation.
+        """
+        flag_value = ""
+        for flag in self.flags:
+            if flag["name"] == "nist_cvss_validation":
+                flag_value = flag["status"]
+
+        mapping = {
+            "": Flaw.FlawNistCvssValidation.NOVALUE,
+            "?": Flaw.FlawNistCvssValidation.REQUESTED,
+            "+": Flaw.FlawNistCvssValidation.APPROVED,
+            "-": Flaw.FlawNistCvssValidation.REJECTED,
+        }
+
+        # If the flag is a different character, let this crash. Shouldn't
+        # happen though, so no special handling. BZ docs specify only ?+-
+        # https://www.bugzilla.org/docs/2.18/html/flags-overview.html
```

Comment:
I am unsure if this note is necessary. As you mentioned, there should be only values `""`/`"?"`/`"+"`/`"-"`, so no need to differentiate between `[]` and `get().`

Reply:
I'll change it to match `requires_summary()`.